ddtrace/internal/encoding.py

```diff
@@ -50,12 +50,51 @@ def encode(self, obj):
         """
         raise NotImplementedError()
 
+    @staticmethod
+    def span_to_dict(span):
```

Comment:
do we want to make this private?

It is internal, but if we ever expose this class public

Reply:
Agreed. Had not considered what would happen if an encoder class became exposed in the public API.

Reply:
I don't expect it to either, but... since you need to use it in tests, using it as a private method doesn't make much sense does it? 🤔